full_snapshot.proto

```diff
@@ -235,6 +236,67 @@ message TablespaceInformation {
   repeated string config = 4;   // Tablespace-level options, as "keyword=value" strings
 }
 
+message CollectorConfig {
+  bool enable_reports = 1;
+  bool disable_logs = 2;
+  bool disable_activity = 3;
+  bool enable_log_explain = 4;
+
+  string db_name = 5;
+  string db_username = 6;
+  string db_host = 7;
+  int32 db_port = 8;
+  string db_sslmode = 9;
+  string db_sslrootcert = 10;
+  string db_sslcert = 11;
+  string db_sslkey = 12;
+
+  repeated string db_extra_names = 13;
+  bool db_all_names = 14;
+
+  string aws_region = 15;
+  string aws_db_instance_id = 16;
+  string aws_access_key_id = 17;
```

Comment:
Whilst this might be interesting for debugging guidance, I think its better if we don't send the key ID.

Reply:
We may want to send something like "has_aws_access_key_id" as a boolean (and the same for secret key), just so we know how somebody has configured it (without knowing the actual value)

Reply:
I like the `has_` pattern. Are there any other sensitive fields you think we should do that for instead of omitting them entirely? Here's the config link https://github.com/pganalyze/collector/blob/master/config/config.go#L27

Reply:
I noted one other case (for the key/cert paths) below, we should be good otherwise.